docs/_includes/date-picker.html

```diff
@@ -0,0 +1,212 @@
+<div id="date-picker">
+    {% include section-header.html name="date-picker" version=page.versions.date-picker %}
+
+    <p>The date picker component is used to display a calendar from which dates can be selected, either as single items or as a range.</p>
+
+    <h3>Single Pane</h3>
+
+    <button class="btn" onclick="document.getElementById('date-picker-example').toggleAttribute('hidden')">Show/Hide</button>
+
+    <div id="date-picker-example" class="date-picker date-picker--single-month" role="dialog" aria-label="date picker">
```

Comment:
Can we demo the date picker outside of a button? Like we do with menu & listbox. I think it should be portable enough to drop into other places.

Reply:
I wonder if `date-picker` and `date-picker-button` would be more apt? Like we do with menu & listbox?

Reply:
I like this idea, I will make the transition in the next commit.